src/main/java/lotto/domain/LottoResult.java

```diff
@@ -25,14 +22,4 @@ public double getProfitRate(LottoPurchaseBudget budget) {
 
         return budget.calculateProfitRate(profit);
```

Comment:
map의 `entrySet`을 이용해보는 방법도 있을 것 같아요. 🙂 
오히려 `getKey`, `getValue`를 해야해서 코드 가독성이 더 안좋아지는 것 같기도하네요. 👀

이 방식이 낫다는 의미가 아니라 상황에 따라서는 entrySet을 활용해볼 수 있다 정도로 봐주세요. 😉

```suggestion
        result.entrySet()
            .stream()
            .mapToLong(entry -> entry.getKey().totalReward(entry.getValue()))
            .sum();
```